acts.sh

```diff
@@ -0,0 +1,33 @@
+package: ACTS
+version: v23.4.0
+build_requires:
+    - "GCC-Toolchain:(?!osx)"
+    - CMake
+    - boost
+source: https://github.com/acts-project/acts.git
+---
+#!/bin/bash -ex
+cmake $SOURCEDIR -DCMAKE_INSTALL_PREFIX=$INSTALLROOT       \
+                 -DCMAKE_BUILD_TYPE=$CMAKE_BUILD_TYPE      \
+                 -DCMAKE_SKIP_RPATH=TRUE
+cmake --build . -- ${JOBS:+-j$JOBS}
+
+#ModuleFile
+mkdir -p etc/modulefiles
+cat > etc/modulefiles/$PKGNAME <<EoF
```

Comment:
```suggestion
MODULEDIR="$INSTALLROOT/etc/modulefiles"
MODULEFILE="$MODULEDIR/$PKGNAME"
mkdir -p "$MODULEDIR"
alibuild-generate-module > "$MODULEFILE"
cat >> "$MODULEFILE" <<EoF
```

Reply:
This fixes the module not found. I think once suggestion is implemented we can remove WIP and ask for approval

Reply:
> Note that the PR is not enough. We also need to have Eigen3 as part of alidist. I am preparing a PR into this PR, and I also fixed the module file generation. No need to do anything.

That's even better! Just for my education: what's the general approach with prerequisites?
I thought that since we do not need (yet) any specific tag/version of Eigen we could have assumed it to be installed outside aliBuild and possibly put it in a recipe later in case we forked in AliceO2Group org.

By the way, would it make sense to also fork ACTS already and refer to a AliceO2Group/ACTS alrady?